charts/icm-as/templates/as-deployment.yaml

```diff
@@ -49,10 +49,6 @@ spec:
         - name: icm-as-server
           image: "{{ .Values.image.repository }}"
           imagePullPolicy: "{{ .Values.image.pullPolicy }}"
-          command:
```

Comment:
We still need this for the ISTE test execution. As far as I know there is no entrypoint field in k8s. So this command would overwrite the ENTRYPOINT in the Dockerfile.